Feature/tests

KartaviK opened the pull request:
Fix fetching `language` attribute for panels
Fix condition for street type in response generating

KartaviK committed:
integrate google-autocomplete
configure composer.json
configure bootstrap

KartaviK committed:
fix code style for streets params rules

KartaviK committed:
edits

KartaviK committed:
few fix tests

KartaviK committed:
remove excess use namespace

KartaviK committed:
preform returning 503 code if service is not work

KartaviK committed:
fix http exception message
add eols

KartaviK committed:
Merge branch 'master' of github.com:wearesho-team/yii2-google-autocomplete into feature/panels

KartaviK committed:
Add tests for panels

KartaviK committed:
refactor tests

KartaviK committed:
remove excess test attribute

KartaviK committed:
remove excess null condition for street type in response generating

Horat1us commented on tests/Unit/Panels/StreetsPanelTest.php:
```diff
@@ -2,27 +2,92 @@
+    {
+        parent::setUp();
+
+        $this->expectedRules = [
```
Rules should not be checked as array equal. Rules should be checked using validation.
So, you need to fill attribute and then run ->validate(['attributeName']) with some error/success expectation.

KartaviK committed:
fix tests
add readme

Horat1us commented on README.md:
````diff
@@ -0,0 +1,43 @@
+];
+```
+
+## Authors
````
Contributors section

Horat1us commented on README.md:
```diff
@@ -0,0 +1,43 @@
+$module = [
+    'controllerMap' => [
+        'autocomplete' => [
+            'class' => \Wearesho\GoogleAutocomplete\Yii\Controller::class,
```
Example should have `use Wearesho\GoogleAutocomplete` in header.

Horat1us commented on README.md:
```diff
@@ -0,0 +1,43 @@
+# Yii2 Google autocomplete api integrate
```
Missing `Installation` section

Horat1us replied:
Titles have to be first-character uppercased in english

KartaviK committed:
Fix readme format:
fix fist-letters case in main title
renamed Authors section to Contributors
add use namespaces to Usage examples

KartaviK committed:
add explicit tests for panels' attributes validation

KartaviK committed:
fix full response test for cities panel
